src/main/java/io/appium/java_client/AppiumClientConfig.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client;
+
+import org.openqa.selenium.remote.http.ClientConfig;
+
+import java.time.Duration;
+
+/**
+ * Manage Appium Client configurations.
+ */
+
+public class AppiumClientConfig {
+    private boolean directConnect;
+
+    private ClientConfig clientConfig;
+
+    private static final Duration DEFAULT_READ_TIMEOUT = Duration.ofMinutes(10);
+
+    public AppiumClientConfig() {
+        this.clientConfig = ClientConfig.defaultConfig().readTimeout(DEFAULT_READ_TIMEOUT);
+        this.directConnect = false;
+    }
+
+    public static class DirectConnect {
```

Comment:
what is the purpose of this static class? variables could be also defined in the main one (don't forget to add "final" though)